build.sh

```diff
@@ -1,6 +1,30 @@
 #!/bin/bash
 # the path to the ComputeCPP package root directory, e.g. /home/user/ComputeCpp-CE-0.1-Linux/
-PACKAGE_ROOT=$1
+
+# How to use build.sh to compile SyclParallelSTL with ComputeCpp ?
+# ./build.sh "path/to/ComputeCpp"
+#
+# How to use build.sh to compile SyclParallelSTL with triSYCL ?
+# ./build.sh --trisycl "path/to/triSYCL" "path/to/compute"
+#
+# paths can be relative ones, this script will make them absoulte before to send them to CMake
+#
+# WARNING/TODO: setting the "path/to/compute" in triSYCL mode is not an option yet
+#
+
+if [ $1 == "--trisycl" ]
+then
+	echo "build.sh enter mode: triSYCL"
+	CMAKE_ARGS="$CMAKE_ARGS -DUSE_COMPUTECPP=OFF -DTRISYCL_PACKAGE_ROOT_DIR=$(readlink -f $2)"
+	if [ $3 ]
```

Comment:
Instead of dealing with this specific code, I would generalize and simplify all this test down to
```
CMAKE_ARGS+=" $@"
```
to pass all the other arguments down to CMake.